cogs/playerstats.py

```diff
@@ -141,6 +174,41 @@ def getAssists(comp):
     table = 'Assist Stats: ' + comp + '\n' + table
     return table
 
+def getInjuries(team="Arsenal"):
+    tableurl = "https://www.premierinjuries.com/injury-table.php"
+    #get table and convert to dataframe
+    header = {
+        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.75 Safari/537.36",
+        "X-Requested-With": "XMLHttpRequest"
+    }
+    injuries = []
+    r = requests.get(tableurl).text
+    bs_obj = BeautifulSoup(r, features="lxml")
+    #currently team_1 is arsenal, but ideally i'd like to have some sort of error-checky way
+    # to make sure that it looks for arsenal and confirms that the team ID is 1.  
+    #i'll let future ren figure that out
+    fullTable = bs_obj.find_all('tr', attrs={'class': "player-row team_1"})
+    playerList = [i.text for i in fullTable]
+    for plObj in playerList:
+        playerDict = {"Player": "", "Reason": "", "FurtherDetail": "", "PotentialReturn": "", "Condition": "", "Status": ""}
+        plObj = plObj.split('\n')
+        plObj.pop(0)
+        player = plObj.pop(0)
+        playerDict["Player"] = player.rsplit("Player")[-1]
+        reason = plObj.pop(0)
+        playerDict["Reason"] = reason.rsplit("Reason")[-1]
+        furtherDetail = plObj.pop(0)
+        playerDict["FurtherDetail"] = furtherDetail.rsplit("Further Detail")[-1]
+        potentialReturn = plObj.pop(0)
+        # dates are in dd/mm/yyyy, AKA the correct way :D
```

Comment:
To avoid date confusion does it make sense to translate to shortform month? AKA 01/10/2022 would simply be Oct. 1 in the table? Would be easy enough with `DateTime`

Reply:
i have made this change as well.